boards/qemu/arc/Kconfig.qemu_arc

```diff
@@ -7,3 +7,4 @@ config BOARD_QEMU_ARC
 	select SOC_QEMU_ARC_HS if BOARD_QEMU_ARC_QEMU_ARC_HS || BOARD_QEMU_ARC_QEMU_ARC_HS_XIP
 	select SOC_QEMU_ARC_HS5X if BOARD_QEMU_ARC_QEMU_ARC_HS5X
 	select SOC_QEMU_ARC_HS6X if BOARD_QEMU_ARC_QEMU_ARC_HS6X
+	select LINKER_ALLOW_RWX_SEGMENTS if !XIP
```

Comment:
this is not qemu/arc specific, so I would enable it by default for everyone.

Reply:
yup.